### ISSUE TITLE
Remove duplicate train-test split cell in CreditCardModel.ipynb

### ISSUE THREAD

**Issue by shubsolos19** [TUFAQUE/CREDIT-CARD-FRAUD-DETECTION-MODEL#6]
Problem

The notebook contain duplicate cell that run same code twice:
Cell 8:
python# Split data
X = df.drop('Class', axis=1)
y = df['Class']
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42, stratify=y)

Cell 9:
pythonX_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42, stratify=y)
Both cell do identical thing. Cell 9 exact duplicate of Cell 8.
Impact

Confusing for reader (why run same code twice?)
Waste computation (redundant)
Bad practice in educational material
Hard to maintain

Notebook run same
Cleaner code
Better for learners

@TUFAQUE  , I would love to work on this issue as part of Nexus Spring of Code 2026. Please assign this to me.

**Comment by algonix01w-ai** [comment]
Hey! @TUFAQUE 
I would like to work on this issue for Nexus Spring of Code 2026. Could you please assign it to me?

**Comment by Raghav7-tech** [comment]
Hey! @TUFAQUE
please assign  this  issue to  me. i would  like to  work  on  this issue.

**Comment by vijayalakshmipokala** [comment]
Hi @TUFAQUE, I’ve reviewed the notebook and confirmed that Cell 9 is indeed a redundant duplicate of Cell 8. I'm ready to clean this up and ensure the notebook flows logically for the Nexus Spring of Code '26. May I be assigned to this
